api/python/cellxgene_census/tests/test_acceptance.py

```diff
@@ -26,6 +26,7 @@ def make_context(census_version: str, config: Optional[Dict[str, Any]] = None) -
     version = cellxgene_census.get_census_version_description(census_version)
     s3_region = version["soma"].get("s3_region", "us-west-2")
     config.update({"vfs.s3.region": s3_region})
+    config.update({"vfs.s3.no_sign_request": "true"})
     return soma.options.SOMATileDBContext(tiledb_ctx=tiledb.Ctx(config))
```

Comment:
slightly off-topic:  this code would be better written:
> return soma.options.SOMATileDBContext(tiledb_config=config)

I.e., remove the `tiledb` dependency.